docs/toml.md

```diff
@@ -803,6 +803,13 @@ domain = "marathon.localhost"
 # Optional
 #
 # dcosToken = "xxxxxx"
+
+# Override DialerTimeout
+#
+# Optional
+# Default: 60
+# dialerTimeout = 5
```

Comment:
Running with 1s or "1s" causes a parse error, as duration essentially is an int64.

I'll be adding to the traefik.sample.toml now.